src/views/GovernanceReviewTeam/Actions/RejectIntake.tsx

```diff
@@ -33,15 +43,21 @@ const RejectIntake = () => {
 
   const onSubmit = (values: RejectIntakeForm) => {
     const { feedback, nextSteps, reason } = values;
-    const rejectPayload = {
-      rejectionNextSteps: nextSteps,
-      rejectionReason: reason,
-      feedback
+
+    const input = {
+      feedback,
+      intakeId: systemId,
+      nextSteps,
+      reason
     };
-    const payload = { id: systemId, rejectPayload };
-    dispatch(rejectSystemIntake(payload));
 
-    history.push(`/governance-review-team/${systemId}/intake-request`);
+    mutate({
+      variables: { input }
+    }).then(response => {
+      if (!response.errors) {
```

Comment:
I think we might need to handle validation errors from the backend, although I think the front end should generally prevent those. They would appear in `response.data.userErrors`.